components/api_request_helper/api_request_helper.cc

```diff
@@ -174,6 +211,7 @@ std::unique_ptr<network::SimpleURLLoader> APIRequestHelper::CreateLoader(
 
 void APIRequestHelper::OnResponse(
     SimpleURLLoaderList::iterator iter,
+    bool sanitize_json,
```

Comment:
it looks like you're trying to use APIRequestHelper to download raw resources and that is not correct. This is specifically for parsing json apis in a safe way